fdbserver/workloads/TagThrottleApi.actor.cpp

```diff
@@ -137,7 +138,8 @@ struct TagThrottleApiWorkload : TestWorkload {
 			}
 		}
 
-		bool removed = wait(ThrottleApi::unthrottleTags(cx, tagSet, throttleType, priority));
+		state Reference<DatabaseContext> db = Reference<DatabaseContext>::addRef(cx.getPtr());
```

Comment:
I think this can be a non-state variable. You could also expose a method on the `Database` to return the `Reference` directly and simplify these usages, if you wanted.

Reply:
Make sense. This should be simplified.
Added the method to expose the Reference and change all the usages.